openbb_terminal/cryptocurrency/overview/overview_controller.py

```diff
@@ -135,7 +135,16 @@ def __init__(self, queue: List[str] = None):
                 "--bar": {},
             }
             choices["categories"] = {
-                "--sortby": {c: {} for c in pycoingecko_model.CATEGORIES_FILTERS},
+                "--sortby": {
+                    c: {}
+                    for c in [
+                        "Name",
+                        "Market_Cap",
+                        "Market_Cap_Change_24H",
+                        "Top_3_Coins",
+                        "Volume_24H",
```

Comment:
Convention is to define this as a self.CATEGORY_SORT_CHOICES then use this here and in the choices